tests/sentry/utils/test_transitional_serializer.py

```diff
@@ -0,0 +1,24 @@
+from django.contrib.sessions.serializers import PickleSerializer
+from django.core.signing import JSONSerializer
+from django.test.testcases import TestCase
+
+from sentry.utils.transitional_serializer import TransitionalSerializer
+
+
+class TransitionalSerializerTest(TestCase):
+    obj = {"id": 1, "user": "user", "email": "email"}
```

Comment:
probably worth having one more test that reads data that isn't json serializable -- like a datetime for example. this test can be removed in the next PR that changes writes.